nvidia_nim/datadog_checks/nvidia_nim/data/conf.yaml.example

```diff
@@ -624,3 +624,23 @@ instances:
     #   - <INCLUDE_REGEX>
     #   exclude:
     #   - <EXCLUDE_REGEX>
+
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event).
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which integration sent the logs.
+## encoding - optional - For file specifies the file encoding. Default is utf-8. Other
+##                       possible values are utf-16-le and utf-16-be.
```

Comment:
```suggestion
##                       possible values are `utf-16-le` and `utf-16-be`.
```

Reply:
All of this is scaffolding that's done as part of syncing the spec. These changes are made automatically so committing them will force the PR to fail. I can suggest changing the spec but as this releases this week I'm going to make the suggested changes in a follow up PR.